website/docs/tutorials/react-native.md

```diff
@@ -23,14 +23,6 @@ React Native does not support all `Intl` features out of the box and we need to
 
 Follow the polyfill installation instructions before proceeding further.
 
-## Metro bundler support
```

Comment:
This is no longer relevant due to the new features in React Native 0.72 related to Metro support, am I right?

Reply:
It was fixed here https://github.com/lingui/js-lingui/issues/1633 🙂